src/BaseOAuth.php

```diff
@@ -230,7 +230,11 @@ protected function sendRequest($request)
         $response = $request->send();
 
         if (!$response->getIsOk()) {
-            throw new InvalidResponseException($response, 'Request failed with code: ' . $response->getStatusCode() . ', message: ' . $response->getContent());
+            throw new InvalidResponseException(
+                $response,
+                'Request failed with code: ' . $response->getStatusCode() . ', message: ' . $response->getContent(),
```

Comment:
You could extract `$response->getStatusCode()` to a variable since you are using it twice now. Also you can cast that variable to int with `(int) $variable` (don't use `intval()`).

Reply:
It's done now, thanks